packages/console/src/pages/Applications/components/LibrarySelector/index.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import React, { useState, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import highFive from '@/assets/images/high-five.svg';
+import tada from '@/assets/images/tada.svg';
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import { SupportedJavascriptLibraries } from '@/types/applications';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  className?: string;
+  libraryName?: SupportedJavascriptLibraries;
+  onChange?: (value: string) => void;
+  onToggle?: () => void;
```

Comment:
nit: do we need to pass the target library name here

Reply:
Yes, the library name is the default value of this selector component.